src/components/FeatureViews/Registration/RegistrationCart/PaymentBoard.js

```diff
@@ -213,21 +296,23 @@ export default function PaymentBoard() {
 	const handlePayment = async () => {
 		const paymentMethod = paymentMethodState.find(({checked}) => checked)?.value;
 		const {data: {enrollments}} = enrollmentResponse;
+		const isSameEnrollment = ({enrollment, course, student}) =>
+			(`${enrollment.student.user.id}-${enrollment.course.id}` === `${student}-${course}`);
```

Comment:
Wouldn't it be clearer to compare `student === enrollment.student.user.id` and `course === enrollment.course.id`?

Reply:
i was desperate. i was trying to figure out why i couldn't filter out existing enrollments to prevent double enrollments. :( i'll change it back though lol